pytest_splunk_addon/standard_lib/event_ingestors/hec_raw_ingestor.py

```
@@ -90,4 +91,5 @@ def __ingest(self, event, params):
 
         except Exception as e:
             LOGGER.error(e)
+            warnings.warn(UserWarning("Error occcured while ingestion: {}".format(e)))
             os._exit(0)
```

warnings don't work if we os.exit(). Please raise an Exception instead.